<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelMixerForm</class>
 <widget class="QWidget" name="ChannelMixerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>147</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="LoadButton">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset resource="../../qrc/photivo.qrc">
         <normaloff>:/dark/icons/document-open.png</normaloff>:/dark/icons/document-open.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="SaveButton">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../../qrc/photivo.qrc">
         <normaloff>:/dark/icons/document-save.png</normaloff>:/dark/icons/document-save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MixernameLabel">
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ChannelMatrixLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="ChannelMatrixGrid">
       <item row="0" column="1">
        <widget class="QLabel" name="RedInLabel">
         <property name="text">
          <string>Red</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="RedOutLabel">
         <property name="text">
          <string>Red</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="BlueInLabel">
         <property name="text">
          <string>Blue</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="ptInput" name="Green2Green"/>
       </item>
       <item row="2" column="3">
        <widget class="ptInput" name="Blue2Red"/>
       </item>
       <item row="5" column="1">
        <widget class="ptInput" name="Red2Blue"/>
       </item>
       <item row="5" column="2">
        <widget class="ptInput" name="Green2Blue"/>
       </item>
       <item row="4" column="1">
        <widget class="ptInput" name="Red2Green"/>
       </item>
       <item row="4" column="3">
        <widget class="ptInput" name="Blue2Green"/>
       </item>
       <item row="5" column="3">
        <widget class="ptInput" name="Blue2Blue"/>
       </item>
       <item row="2" column="1">
        <widget class="ptInput" name="Red2Red"/>
       </item>
       <item row="2" column="2">
        <widget class="ptInput" name="Green2Red"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="GreenInLabel">
         <property name="text">
          <string>Green</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="BlueOutLabel">
         <property name="text">
          <string>Blue</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="GreenOutLabel">
         <property name="text">
          <string>Green</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ptInput</class>
   <extends>QProgressBar</extends>
   <header>ptInput.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../qrc/photivo.qrc"/>
 </resources>
 <connections/>
</ui>
